2fa_tool/__init__.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+
+"""Generates 2 step verification (2FA) codes.
+
+Save your key in a file located in "~/.2fa/.key", then it will work.
+
+Save your key:
+$ mkdir ~/.2fa
+$ echo 'your key' > ~/.2fa/.key
+
+Synopsis: <trigger> """
+
+import base64
+import hashlib
+import hmac
+import os
+import struct
+import time
+
+import six
+from albert import *
+
+__title__ = "2FA Tool"
+__version__ = "0.1.1"
+__triggers__ = "2fa "
+__authors__ = "geeknonerd"
+
+md_iid = "0.2"
```

Comment:
this will not work. the curren interface id is 0.5 (https://github.com/albertlauncher/plugins/tree/master/python#plugin-metadata) have you actually tested this plugin?